src/lib/server/generateStaticHTMLFile.js

```diff
@@ -0,0 +1,33 @@
+import path from "path";
+import { prepareOutput } from "./RequestHandler";
+import { getLogger } from "./logger";
+import WebpackIsomorphicTools from "webpack-isomorphic-tools";
+import webpackConfig from "../../config/webpack-isomorphic-tools-config";
+import detectEnvironmentVariables from "../detectEnvironmentVariables";
+
+const LOGGER = getLogger();
+
+export default async function () {
+  const mockReq = {
+    headers: {
+      host: "localhost"
+    },
+    url: "/"
+  };
+
+  LOGGER.debug("about to run middleware");
+  global.webpackIsomorphicTools = new WebpackIsomorphicTools(webpackConfig)
```

Comment:
the prepareOutput function relies on `webpackIsomorphicTools`. You can view more information about what that is here: https://github.com/halt-hammerzeit/webpack-isomorphic-tools but just know that the method needs to be run inside of the callback after configuring it.